content/en/docs/gke/customizing-gke.md

```diff
@@ -146,6 +146,42 @@ For example, to modify settings for the Jupyter web app:
 
 ## Common customizations
 
+### Add users to Kubeflow
+
+You must grant each user the minimal permission scope that allows them to 
+connect to the Kubernetes cluster.
+
+For Google Cloud, you should grant the following Cloud Identity and Access Management (IAM) roles.
+
+In the following commands, replace `[PROJECT]` with your Google Cloud project and replace `[EMAIL]` with the user's email address:
+
+* To access the Kubernetes cluster, the user needs the [Kubernetes Engine 
+  Cluster Viewer](https://cloud.google.com/kubernetes-engine/docs/how-to/iam)
+  role:
+
+    ```
+    gcloud projects add-iam-policy-binding [PROJECT] --member=user:[EMAIL] --role=roles/container.clusterViewer
+    ```
+
+* To access the Kubeflow UI through IAP, the user needs the
+  [IAP-secured Web App User](https://cloud.google.com/iap/docs/managing-access)
+  role:
+
+    ```
+    gcloud projects add-iam-policy-binding [PROJECT] --member=user:[EMAIL] --role=roles/iap.httpsResourceAccessor
+    ```
+
+    Note, you need to grant the user `IAP-secured Web App User` role even if the user is already an owner or editor of the project. `IAP-secured Web App User` role is not implied by just `Project Owner` or `Project Editor` roles.
+
+* To be able to run `gcloud get credentials` and see logs in Cloud Logging
+  (formerly Stackdriver), the user needs viewer access on the project:
+
+    ```
+    gcloud projects add-iam-policy-binding [PROJECT] --member=user:[EMAIL] --role=roles/viewer
+    ```
+
+Alternatively, you can also grant these roles on the [IAM page in the Cloud Console](https://console.cloud.google.com/iam-admin/iam). Make sure you are in the same project as your Kubeflow deployment.
+
 <a id="gpu-config"></a>
 ### Add GPU nodes to your cluster
 
```

Comment:
Line 191 (sorry, I couldn't select it)

- For consistency: `Google Cloud` ~~`Platform`~~/~~`(GCP)`~~, as you did in the other file.